configuration/ccdaReferenceValidatorConfig.xml

```diff
@@ -2571,7 +2571,7 @@
 		<validator>
 			<name>ValueSetCodeValidator</name>
 			<validationResultSeverityLevels>
-				<codeSeverityLevel>SHOULD</codeSeverityLevel>
+				<codeSeverityLevel>SHALL</codeSeverityLevel>
```

Comment:
Should be MAY unless the cert is overriding, is it?
Need to verify others as well.